arcjet-next/index.ts

```diff
@@ -327,7 +330,20 @@ export default function arcjet<
           ExtraProps<Rules>
         >;
 
-        return aj.protect({}, req);
+        const getBody = async () => {
+          try {
+            if (request.clone !== undefined) {
+              const cloned = request.clone();
+              return await cloned.text();
+            } else if (request.body !== undefined) {
+              return JSON.stringify(request.body);
```

Comment:
@e-moran this needs an answer

Reply:
Yeah it is more Node.js-like, but they have a few helpers such as `.body`

https://nextjs.org/docs/pages/building-your-application/routing/api-routes#request-helpers

Reply:
I wonder if we need a pages route in your next example so we can test this stuff 🤔

Reply:
I'm also just realizing that this probably needs `readBody` as well because Next users can disable body parsing. I think we should merge this PR and open an issue to build a couple pages examples and implement `readBody`.

Reply:
https://github.com/arcjet/arcjet-js/issues/1399